extension/fts/src/function/query_fts_gds.cpp

```diff
@@ -315,19 +312,34 @@ binder::expression_vector QFTSAlgorithm::getResultColumns(binder::Binder* binder
 }
 
 void QFTSAlgorithm::bind(const GDSBindInput& input, main::ClientContext& context) {
-    KU_ASSERT(input.getNumParams() == 9);
-    auto termNode = input.getParam(1);
-    auto k = input.getLiteralVal<double>(2);
-    auto b = input.getLiteralVal<double>(3);
-    auto numDocs = input.getLiteralVal<uint64_t>(4);
-    auto avgDocLen = input.getLiteralVal<double>(5);
-    auto numTermsInQuery = input.getLiteralVal<uint64_t>(6);
-    auto isConjunctive = input.getLiteralVal<bool>(7);
-    auto inputTableName = input.getLiteralVal<std::string>(8);
-    auto entry = context.getCatalog()->getTableCatalogEntry(context.getTx(), inputTableName);
-    auto nodeOutput = bindNodeOutput(input.binder, {entry});
-    bindData = std::make_unique<QFTSGDSBindData>(termNode, nodeOutput, k, b, numDocs, avgDocLen,
-        numTermsInQuery, isConjunctive);
+    KU_ASSERT(input.getNumParams() == 10);
+    // Bind graph entry.
+    auto catalog = context.getCatalog();
+    auto tx = context.getTx();
+    auto tableName = ExpressionUtil::getLiteralValue<std::string>(*input.getParam(8));
+    auto indexName = ExpressionUtil::getLiteralValue<std::string>(*input.getParam(9));
+    auto tableEntry = catalog->getTableCatalogEntry(tx, tableName);
+    auto termsTableName = FTSUtils::getTermsTableName(tableEntry->getTableID(), indexName);
+    auto docsTableName = FTSUtils::getDocsTableName(tableEntry->getTableID(), indexName);
+    auto appearsInTableName = FTSUtils::getAppearsInTableName(tableEntry->getTableID(), indexName);
+    auto termsEntry = catalog->getTableCatalogEntry(tx, termsTableName);
+    auto docsEntry = catalog->getTableCatalogEntry(tx, docsTableName);
+    auto appearsInEntry = catalog->getTableCatalogEntry(tx, appearsInTableName);
+    auto graphEntry = graph::GraphEntry({termsEntry, docsEntry}, {appearsInEntry});
+    // Bind output node.
+    auto nodeOutput = bindNodeOutput(input.binder, {tableEntry});
+    auto qftsBindData = std::make_unique<QFTSGDSBindData>(std::move(graphEntry), nodeOutput);
```

Comment:
Can we not save the graphEntry in the bindData? I think we can generate the graphEntry during edgeCompute